include/nighthawk/common/factories.h

```diff
@@ -24,7 +24,7 @@ class SequencerFactory {
                               const SequencerTarget& sequencer_target,
                               TerminationPredicatePtr&& termination_predicate,
                               Envoy::Stats::Scope& scope,
-                              const Envoy::SystemTime scheduled_starting_time) const PURE;
+                              const Envoy::MonotonicTime scheduled_starting_time) const PURE;
```

Comment:
Not actionable, just making sure I'm understanding - this is where we're moving back to MonotonicTime to prevent #569 ?

Reply:
Yes, that is right.